src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @Test
+    void split_test() {
+        String str = "1,2";
+        String[] strArr = str.split(",");
```

Comment:
```suggestion
        String[] strArr = "1,2".split(",");
```

로컬 변수 없이 위와 같이 구현하는 것은 어떨까?